README.md

```diff
@@ -82,6 +83,29 @@ Which will load the following `<script>` asynchronously:
 
 View the [full list of supported script parameters](https://developer.paypal.com/docs/business/javascript-sdk/javascript-sdk-configuration/#script-parameters).
 
+### Legacy Browser Support
+
+This library relies on [JavaScript Promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise). To support legacy browsers like IE 11, you must provide your own Promise polyfill. With a Promise polyfill this library will support the same browsers as the JS SDK. Here's the [full list of supported browsers](https://developer.paypal.com/docs/business/checkout/reference/browser-support/#supported-browsers-by-platform).
```

Comment:
Nit-picking, but I think it could be made clearer that the "full list of supported browsers" that is provided is explicitly referring to the list of browsers that the full JS SDK supports. Maybe not having that link live in separate sentence? 
>This library relies on [JavaScript Promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise). To support legacy browsers like IE 11, you must provide your own Promise polyfill. With a Promise polyfill this library will [support the same browsers](https://developer.paypal.com/docs/business/checkout/reference/browser-support/#supported-browsers-by-platform) as the JS SDK.

Reply:
@BriceShatzer thanks for the suggestion. That is defnitely better. I just pushed up the change.